README.md

```diff
@@ -25,19 +33,19 @@ directory, and makes it possible to "git update" the library later as it gets im
 (Things can go wrong here, and you might need "pip3" or "sudo pip" if you have 
```

Comment:
you should remove the `pip install -e ` command.

In fact this whole section is kinda pointless now there's a pypi version